src/components/Footer.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import './Footer.css';
+
+export default function Footer() {
+
+  const dispatch = useDispatch();
+
+  const youText = React.createRef();
+
+  const addTask = () =>{
+
+    if(youText.current.value) {
+      dispatch({type: 'ADD TASK', task: {text: youText.current.value, id: Date.now(), isChecked: false}});
+      youText.current.value = '';
+    }
+
+  }
+
+ 
+  
+
+
+  return (
```

Comment:
here, too 
too many empty lines, please, try to use max 1-2 lines to separate the logic

Reply:
Ok, Bro)